pages/blog/tag/[tag].js

```diff
@@ -6,36 +6,27 @@ import MainLayout from "@layouts/main";
 import ContentWrapper from "@components/ContentWrapper";
 
 export default function PostWrapper(props) {
-  const { post, preview, posts } = props;
+  const { preview, posts } = props;
 
   return (
     <MainLayout preview={preview}>
-      <PageMeta
-        title={post.title}
-        description={post.excerpt}
-        url={`${Config.pageMeta.blogIndex.url}/${post.slug}`}
-        canonical={post.externalUrl ? post.externalUrl : false}
-      />
+      <PageMeta title={"Tag"} description={""} url={""} canonical={false} />
       <ContentWrapper>
-        <Post post={post} />
+        {posts.map((post) => (
+          <Post key={post.id} post={post} />
+        ))}
       </ContentWrapper>
     </MainLayout>
   );
 }
 
 export async function getStaticPaths() {
-  // Array<string>
   const blogPostTags = await ContentfulApi.getAllUniquePostTags();
 
-  console.log(blogPostTags);
-  console.log(blogPostTags[0].id);
-
-  const paths = blogPostTags.map((id) => {
-    return { params: { tag: id.toString() } };
+  const paths = blogPostTags.map(({ id }) => {
```

Comment:
@jazzybuilds This was not working because `id` was an object so if we destructure the object we can get just `id`.